modules/identity/src/Volo.Abp.Identity.MongoDB/Volo/Abp/Identity/MongoDB/MongoOrganizationUnitRepository.cs

```diff
@@ -134,5 +139,25 @@ public class MongoOrganizationUnitRepository : MongoDbRepository<IAbpIdentityMon
                 .As<IMongoQueryable<IdentityUser>>()
                 .CountAsync(GetCancellationToken(cancellationToken));
         }
+
+        public virtual Task RemoveAllRolesAsync(OrganizationUnit organizationUnit, CancellationToken cancellationToken = default)
```

Comment:
No need to create such a method while we can directly call organizationUnit.Roles.Clear(); in the upper layer (domain or app service)